lib/ldp_sync.h

```diff
@@ -56,6 +56,11 @@ struct ldp_igp_sync_if_state {
 	bool sync_start;
 };
 
+struct ldp_igp_configure_if_mpls {
+	ifindex_t ifindex;
+	bool status;
```

Comment:
if the status is enabled/disabled, then it's more clear it would be named `bool enabled` (IMO)... Because status could be more than enabled/disabled, right?

Reply:
changed to enabled.